src/tools/pub/cmd/index.md

```diff
@@ -1,132 +1,152 @@
 ---
 title: The pub tool
```

Comment:
I suggest s/The pub tool/The pub client/ now that it's no longer a standalone tool.

Reply:
OK